AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -1,16 +1,17 @@
+#import "MSAnalytics+Validation.h"
```

Comment:
I think the correct order is below ` #import "MSAnalytics.h"`. The rule of sorting import isn't ascii order if I remember correctly.